testcases/OpTestRTCdriver.py

```diff
@@ -115,14 +74,32 @@ def test_RTC_driver(self):
                 continue
         print l_list
 
+
+    ##
+    # @brief This function will cover following test steps
+    #        1. Getting host information(OS and Kernel info)
+    #        2. Loading rtc_opal module based on config option
+    #        3. Testing the rtc driver functions
+    #                Display the current time,
+    #                set the Hardware Clock to a specified time,
+    #                set the Hardware Clock from the System Time, or
+    #                set the System Time from the Hardware Clock
+    #                keep the Hardware clock in UTC or local time format
+    #                Hardware clock compare, predict and adjust functions
+    #                Hardware clock debug and test modes
+    #                Reading the Hardware clock from special file instead of default
+    #        4. After executing above each function reading the Hardware clock in b/w functions.
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def runTest(self):
+        self.test_init()
```

Comment:
After changing test_init to some other name Ex: rtc_init it is working.
But we have new error, l_list should be global to use it in runTest
runTest (testcases.OpTestRTCdriver.FullRTC) ... OpTestSystem START STATE: 6
OpTestSystem TRANSITIONED TO: 6

hwclock from util-linux 2.27.1
0

4.8.0-32-generic
Config file is available
cat /boot/config-4.8.0-32-generic | grep -i --color=never CONFIG_RTC_DRV_OPAL

CONFIG_RTC_DRV_OPAL=y

Driver built into kernel itself

rtc
rtc0

['/dev/rtc', '/dev/rtc0']
ERROR

======================================================================
ERROR: runTest (testcases.OpTestRTCdriver.FullRTC)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestRTCdriver.py", line 98, in runTest
    for l_file in l_list:
NameError: global name 'l_list' is not defined

----------------------------------------------------------------------
Ran 1 test in 31.649s

FAILED (errors=1)